pkg/flags/flags.go

```diff
@@ -34,6 +34,12 @@ func setLogLevel(logLevel string) {
 		log.Fatalf("invalid log-level: %s", logLevel)
 	}
 	log.SetLevel(level)
+
+	// configure klog to be very verbose to stderr when debug logs are requested
+	if level == log.DebugLevel {
+		flag.Set("v", "3")
```

Comment:
i'm not sure what level 3 means.

experimenting a bit with `kubectl`:
```bash
$ kubectl options|grep "log level"
  -v, --v=0: log level for V logs
```
kubectl provides additional info up to level 9:
```bash
$ diff <(kubectl version -v 8 2>&1 | wc -l) <(kubectl version -v 9 2>&1 | wc -l)
1c1
<       23
---
>       20
```

Some detail on these log levels:
https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md

I'd love to hook into some proper constants for this, and mapping our log levels to theirs would be ideal.